web/frontend/angular/package.json

```diff
@@ -25,10 +26,18 @@
   "devDependencies": {
     "@angular-builders/custom-webpack": "^12.1.3",
     "@angular-devkit/build-angular": "~12.2.11",
+    "@angular-eslint/builder": "12.7.0",
+    "@angular-eslint/eslint-plugin": "12.7.0",
+    "@angular-eslint/eslint-plugin-template": "12.7.0",
+    "@angular-eslint/schematics": "12.7.0",
+    "@angular-eslint/template-parser": "12.7.0",
     "@angular/cli": "~12.2.11",
     "@angular/compiler-cli": "~12.2.0",
     "@types/jasmine": "~3.8.0",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
+    "@typescript-eslint/parser": "4.28.2",
+    "eslint": "^7.26.0",
```

Comment:
I don't think any of this has to be installed here. Linting will _always_ run through pre-commit. pre-commit creates its own environment for node. Thus, you only need to specify dependencies in the pre-commit config.